Objects/capsule.c

```diff
@@ -279,6 +303,29 @@ capsule_repr(PyObject *o)
 }
 
 
+static int
+capsule_traverse(PyCapsule *capsule, visitproc visit, void *arg)
```

Comment:
Why not:

```suggestion
capsule_traverse(PyObject *capsule, visitproc visit, void *arg)
```

Reply:
It's convenient to access capsule member in the  code. It's commonly used in Python code base for that. Or do you ask me to do the cast in the function with a separated variable?

Reply:
My bad; of course you need the object layout. Sorry about the noise!